tests/examples_arguments_syntax/interactive_reorder_stacked_bars.py

```diff
@@ -0,0 +1,30 @@
+"""
+Reorder stacked bar segments
+============================
+This example uses a calculate transform
+to check the values of the "site" column
+vs the clicked values in the legend,
+and assigns a lower order (0)
+if there is a match.
+The use of "indexOf" checks for equality in an array,
+which here allows for mutliple segments to be reordered
+by holding down the shift key while clicking the legend.
+"""
+# category: interactive charts
+import altair as alt
+from vega_datasets import data
+
+
+selection = alt.selection_point(fields=['site'], bind='legend')
+
+alt.Chart(data.barley.url).mark_bar().transform_calculate(
```

Comment:
```suggestion
selection = alt.selection_point(fields=['site'], bind='legend')

source = data.barley.url

alt.Chart(source).mark_bar().transform_calculate(
```
To follow the instructions in `CONTRIBUTING.md`:

> For consistency all data used for a visualization should be assigned to the
  variable `source`. Then `source` is passed to the `alt.Chart` object.
  If the example requires multiple dataframes then this does not apply. See
  other examples for guidance.